### PR TITLE
Update Psycys > Pentera

### PR DESCRIPTION
**Pull request by mchasepan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by mchasepan** [comment]
Submitting on behalf of partner. @ShahafBenYakir FYI

File: Packs/Pcysys/pack_metadata.json [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
     "useCases": [],
     "keywords": [],
-    "githubUser": [
-        "TalPcysys"
-    ],
-    "certification": "certified"
-}
\ No newline at end of file
+    "certification": "certified",
+    "authorImage": "content/Packs/Pcysys/Integrations/Pcysys/Pcysys_image.png"
```
Comment (Bargenish) [inline Packs/Pcysys/pack_metadata.json:15]: Do they have a new Github user they can add?

File: Packs/Pcysys/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Updated for new Pentera branding
```
Comment (Bargenish) [inline Packs/Pcysys/ReleaseNotes/1_1_0.md:1]: @ShahafBenYakir Do we have a convention or a header for pack changes?

**Review by Bargenish — APPROVED** [review]
@mchasepan I Pushed a couple of changes to fix the build errors.

File: Packs/Pcysys/ReleaseNotes/1_1_0.md (hunk 2 above)
Comment (ShahafBenYakir) [reply Packs/Pcysys/ReleaseNotes/1_1_0.md:1]: @mchasepan Have we added version and RN on purpose? You want the user to get an "update available" notification?
No convention

Comment (mchasepan) [reply Packs/Pcysys/ReleaseNotes/1_1_0.md:1]: Yes, since its a name change.

File: Packs/Pcysys/pack_metadata.json (hunk 1 above)
Comment (mchasepan) [reply Packs/Pcysys/pack_metadata.json:15]: If no user, it will default to support email address no @Bargenish ? Lets do that.

**Pull request merged**